UICatalog/UICatalog.cs

```diff
@@ -371,13 +371,31 @@ public UICatalogTopLevel ()
 				 * we just measure all the data ourselves and set the appropriate
 				 * max widths as ColumnStyles 
 				 */
-
 				var longestName = _scenarios!.Max (s => s.GetName ().Length);
 				ScenarioList.Style.ColumnStyles.Add (0, new ColumnStyle () { MaxWidth = longestName, MinWidth = longestName, MinAcceptableWidth = longestName });
-				ScenarioList.Style.ColumnStyles.Add (1, new ColumnStyle () {  MaxWidth = 1 });
-
+				ScenarioList.Style.ColumnStyles.Add (1, new ColumnStyle () { MaxWidth = 1 });
+
+				// Enable user to find & select a scenario by typing text
+				// TableView does not (currently) have built-in CollectionNavigator support (the ability for the 
+				// user to type and the items that match get selected). We implement it in the app instead. 
+				ScenarioList.KeyDown += (s, a) => {
+					if (CollectionNavigator.IsCompatibleKey (a.KeyEvent)) {
+						var newItem = _scenarioCollectionNav?.GetNextMatchingItem (ScenarioList.SelectedRow, (char)a.KeyEvent.KeyValue);
+						if (newItem is int && newItem != -1) {
+							ScenarioList.SelectedRow = (int)newItem;
+							ScenarioList.EnsureSelectedCellIsVisible ();
+							ScenarioList.SetNeedsDisplay ();
+							a.Handled = true;
+						}
+					}
+				};
 				ScenarioList.CellActivated += ScenarioView_OpenSelectedItem;
 
+				// TableView typically is a grid where nav keys are biased for moving left/right.
+				ScenarioList.AddKeyBinding (Key.Home, Command.TopHome);
+				ScenarioList.AddKeyBinding (Key.End, Command.BottomEnd);
+				ScenarioList.ClearKeyBinding (Key.A | Key.CtrlMask);
```

Comment:
You can just turn `MultiSelect=false` to prevent this, you don't have to remove the Ctrl+A key.  

Also as an asside you can also multi select with Shift+Up/Down and also Ctrl+Click (if OS supports it).

Reply:
Setting `MultiSelect = false` does not undo the `AddKeyBinding(ctrl-a)`:

```cs
		/// <summary>
		/// True to allow regions to be selected 
		/// </summary>
		/// <value></value>
		public bool MultiSelect { get; set; } = true;
```

I think it would be reasonable for TableView to adapt it's key bindings based on whether `MutlSelect` is on or not; For now I'll just have UI Catalog clear the binding for Ctrl-A.

Reply:
To be clear: the reason is Ctrl-A in UI Catalog pops the about box. It could use a different shortcut, but I have muscle memory and frequently test the about box to ensure stuff is working.